server/api/tests.js

```diff
@@ -161,4 +159,19 @@ router.get('/results', async (req, res, next) => {
   }
 });
 
+router.get('/jest', async (req, res, next) => {
+  try {
+    const util = require('util');
+    const exec = util.promisify(require('child_process').exec);
+    const { stdout, stderr } = await exec('npm run test sum.test.js');
+    console.log('stdout:', stdout);
+    console.log('stderr:', stderr);
+
+    res.send(stderr);
+  } catch (err) {
+    res.send(err);
+    next(err);
+  }
+});

```

Comment:
Attempting this
https://stackoverflow.com/a/56095793
```js
const util = require('util');
const exec = util.promisify(require('child_process').exec);

async function lsExample() {
  try {
    const { stdout, stderr } = await exec('ls');
    console.log('stdout:', stdout);
    console.log('stderr:', stderr);
  } catch (e) {
    console.error(e); // should contain code (exit code) and signal (that caused the termination).
  }
}
lsExample()
```

Reply:
Shows on the frontend
<img width="378" alt="Screen Shot 2022-09-21 at 7 31 17 PM" src="https://user-images.githubusercontent.com/4698973/191627723-e78ecd03-1401-4b33-ba46-ff98c3b22ccb.png">

Reply:
<img width="378" alt="Screen Shot 2022-09-21 at 7 31 17 PM" src="https://user-images.githubusercontent.com/4698973/191629383-2ef66c4f-262e-47ad-911e-485650db3221.png">